trello/features/smoke/SmokeBoard.feature

```diff
@@ -13,3 +13,18 @@ Feature: Board's Test
     And Sets a DELETE request to "/boards/(idBoard)"
     And Sends request
     And Should return status code 200
+
+    # Create by Limbert Vargas at 1/28/2020
+  Scenario: Deletes a board by Id
+    When Sets a DELETE request to /boards/(id) endpoint
```

Comment:
change this Sets a DELETE request to /boards/(id) endpoint to Sets a DELETE request to "/boards/(id)"